fleetspeak/src/inttesting/frr/proto/fleetspeak_frr/frr.proto

```diff
@@ -92,4 +105,14 @@ service Master {
   // by the FS server.
   rpc RecordFileResponse(FileResponseInfo) returns (fleetspeak.EmptyMessage) {
   }
+
+  // CompletedRequests returns a list of requests made to a client which have been
+  // completed.
+  rpc CompletedRequests(CompletedRequestsRequest) returns (CompletedRequestsResponse) {
+  }
+
+  // CreateHunt initiates a hunt which sends the provided TrafficRequestData to
+  // every client, up to limit.
+  rpc CreateHunt(CreateHuntRequest) returns (fleetspeak.EmptyMessage) {
```

Comment:
On second thought, let's make this return a CreateHuntResponse, which will be just an empty message. (For extensibility).